### ISSUE TITLE
sh: error importing function definition

### ISSUE THREAD

**Issue by advorzhak** [platformio/platformio-atom-ide#1729]
PIP: sh: mc: line 1: syntax error: unexpected end of file
sh: error importing function definition for `mc'
Could not import setuptools which is required to install from a source distribution.
sh: mc: line 1: syntax error: unexpected end of file
sh: error importing function definition for `mc'

**Comment by ivankravets** [comment]
Please provide an output from a system Terminal
```
echo $PATH
echo $PYTHONPATH
python --version
pip --version
pip list
```

**Comment by ivankravets** [comment]
We have just released a new version [2.7.0](https://github.com/platformio/platformio-atom-ide/releases/tag/v2.7.0), please upgrade.

**Please note that we HIGHLY recommend moving to [PlatformIO IDE for VSCode](https://platformio.org/platformio-ide) which resolves tons of issues and brings a lot of new features.**
